src/backend/infrastructure/grant-adapp-sql-access.ps1

```diff
@@ -4,6 +4,8 @@ param(
     [string]$sqlDatabaseName
 )
 
+Write-Host "Installing SQLServer module for CurrentUser"
+Install-Module -Name SqlServer -RequiredVersion 21.1.18256  -Force -Scope CurrentUser
```

Comment:
Do you need to specify the version? 
Should add a step to check if the module exists or not - so if the module comes preinstalled we don't waste time installing a new specific version